pkgs/development/libraries/wolfssl/default.nix

```diff
@@ -5,37 +5,55 @@
 , autoreconfHook
 , util-linux
 , openssl
+, variant ? "all"
+, extraFlags ? [ "--enable-pkcs11" "--enable-writedup" ]
+, enableLto ? true
 }:
-
-stdenv.mkDerivation rec {
-  pname = "wolfssl";
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wolfssl-${variant}";
   version = "5.6.3";
 
   src = fetchFromGitHub {
     owner = "wolfSSL";
     repo = "wolfssl";
-    rev = "refs/tags/v${version}-stable";
+    rev = "refs/tags/v${finalAttrs.version}-stable";
     hash = "sha256-UN4zs+Rxh/bsLD1BQA+f1YN/UOJ6OB2HduhoetEp10Y=";
```

Comment:
using finalAttrs here will only hide when the FOD is invalid for the current version

Reply:
Not sure what FOD is, but overriding the version would require a changed hash, so that makes no sense, yeah.